### DISCUSSION TITLE
Student Benefit APPROVED in Sep. 1st ,BUT Haven't be available until now.

### DISCUSSION THREAD

**Discussion by Tohsaka-0** [community/community#138031] [GitHub Education]
### Hi everyone,

<img width="333" alt="Snipaste_2024-09-07_20-17-38" src="https://github.com/user-attachments/assets/a7157b6e-b30d-4d54-92a0-3061a7d33c4d">

Due to the FAQ:
" If you have not received your education benefits **within 5 days** of your approval status please **raise a discussion** on the [GitHub Education community](https://github.com/orgs/community/discussions/categories/github-education) and the Education community team will be able to assist in resolving this for you." 

Asking for help from the community team as instructions.🌟 

My request has been approved in Sep. 1st and today is Sep.7th.

It takes so long. And If there anything I can do to get my student benefit ASAP? I didn't expect this situation and now having problem with my course since I need to utilize some source free for students.

**Comment by davevad93** [comment]
Hi @Tohsaka-0 ,

Yes, it can take a while to obtain the benefits, the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Tohsaka-0 was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by andrefqms** [comment]
If the problem persists I'd suggest you launch a ticket over to Github Education's Support regarding this matter. You can do so [here](https://support.github.com/contact/education).
